README.md

```diff
@@ -16,6 +16,21 @@ Installation
 
 `composer require elife/content-negotiator`
 
+Set up
+------
+
+### Silex
+
+```php
+use eLife\ContentNegotiator\Silex\ContentNegotiationProvider;
```

Comment:
the example could benefit from a `use` for `Accept` to understand where it does come from (`Response` seems obvious but maybe it's not)

Reply:
Added details.